### PR TITLE
Replace blackcaldrons surround.nvim with a working version

### PR DESCRIPTION
**Pull request by sharflare**
The original suuround.vim was not working, so i replaced it with a fork that works perfectly

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -34,7 +34,7 @@ return require('packer').startup(function()
     requires = { {'nvim-lua/plenary.nvim'} }
   }
   use {
-    "blackCauldron7/surround.nvim",
+    "ur4ltz/surround.nvim,
```
Comment (albingroen) [inline init.lua:37]: You're missing a double-quote at the end of the line

Comment (sharflare) [reply init.lua:37]: Oh yea sorry about that, I'm not on my pc rn but when I get on I'll add it

**Comment by albingroen** [comment]
Interesting. I read up on this a little bit, and it seems the author of the `blackCauldron7/surround.nvim` plugin has deleted the repository for it. Good catch and thanks for the update!

File: init.lua (hunk 1 above)
Comment (albingroen) [reply init.lua:37]: @Infinitybeond1 Do you plan on adding the missing double quote any time soon or should I do it?

**Comment by technicalpickles** [comment]
I ran into this as well

**Comment by albingroen** [comment]
I fixed this here https://github.com/albingroen/quick.nvim/pull/9

**Pull request closed without merging**